duckduckgo/Android
# Connect flow UI

Task/Issue URL: https://app.asana.com/0/488551667048375/1204259168623409/f 

### Description
Implements UI for connecting flow.

### Steps to test this PR
(Disclaimer: to test the feature, please remove `FakeDeviceSyncState` to make the option appear in Settings)

For the tests you will need 2 devices here.

_Feature 1_
- [x] Install the app on 2 devices
- [x] on device A: go to settings -> sync -> enable toggle -> Turn on sync -> Sync Another device -> Show QR code
- [x] it will start polling data every 5 seconds
- [x] on device B: go to settings -> sync -> enable toggle -> Turn on sync -> Sync Another device -> Read QR code
- [x] read QR from device A
- [x] It will create an account
- [x] device B should join the account soon
- [x] both devices will show screen with connected device success message


### UI changes
| Before  | After |
| ------ | ----- |
!(Upload before screenshot)|(Upload after screenshot)|
